packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -135,7 +136,7 @@ class _NestedScrollViewState extends State<NestedScrollView> {
     return new CustomScrollView(
       scrollDirection: widget.scrollDirection,
       reverse: widget.reverse,
-      physics: new ClampingScrollPhysics(parent: widget.physics),
+      physics: widget.physics ?? const ClampingScrollPhysics(),
```

Comment:
Thinking about it, a better fix (which would allow us to skip all the stuff about forcing the user to use ClampingScrollPhysics et al) would be to just use `const ClampingScrollPhysics().applyTo(physics)` in the case where `physics` is non-null.